### ISSUE TITLE
question about fixing the contrast

### ISSUE THREAD

**Issue by liu00635** [liu00635/accessible-me#1]
@abhaysharma Hello professor, I tried to improve the contrast by changing the background color but not sure. Could you please help me see if this is enough? Or is there anything I need to add? Thank you!

**Comment by abhaysharma** [comment]
@liu00635 yes, the contrast is good. You can check it by running the wave extension on your page.

**Comment by liu00635** [comment]
Thank you so much!
________________________________
